packages/ui/src/components/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -18,74 +18,71 @@ import {
   Thead,
   Tr,
 } from "@patternfly/react-table";
-import type { FunctionComponent } from "react";
+import { useCallback, useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import type { Consumer } from "ui-models/src/models/consumer-group";
 import "../ConsumerGroup.css";
 import type { OffsetValue } from "../types";
 import { OffsetSelect } from "./OffsetSelect";
 import { TopicSelect } from "./TopicSelect";
+import type { ConsumerGroupState } from "@rhoas/app-services-ui-components";
+import type { FunctionComponent } from "react";
 
 export type ConsumerRow = Consumer & {
   selected?: boolean;
 };
 
 export type ConsumerGroupResetOffsetProps = {
   isModalOpen: boolean;
-  isDisconnected: boolean;
+  state: ConsumerGroupState;
   groupId: string;
   topics: string[];
-  selectedTopic: string;
-  customOffsetValue: string;
-  setcustomOffsetValue: (customOffsetValue: string) => void;
-  consumers: ConsumerRow[];
-  isSelected: boolean;
+  consumers: Consumer[];
   onClickClose: () => void;
-  onClickResetOffset: () => void;
-  onChangeTopic: (value: string) => void;
-  selectedOffset: OffsetValue;
-  onChangeOffsetValue: (value: OffsetValue) => void;
-  confirmCheckboxChecked: boolean;
-  onConfirmationChange: (value: boolean) => void;
-  SelectAllConsumer: (consumer: ConsumerRow[]) => void;
-  onSelectAll: (isSelected: boolean) => void;
-  onSelectRow: (index: number, isSelected: boolean) => void;
+  onClickResetOffset: (
+    topic: string,
+    offset: OffsetValue,
+    partitions: number[],
+    customOffset: string
+  ) => void;
   disableFocusTrap?: boolean;
   appendTo?: () => HTMLElement;
 };
 
 export const ConsumerGroupResetOffset: FunctionComponent<
   ConsumerGroupResetOffsetProps
 > = ({
-  isDisconnected,
   groupId,
   topics,
-  customOffsetValue,
-  setcustomOffsetValue,
-  consumers,
   onClickClose,
   isModalOpen,
   onClickResetOffset,
-  selectedTopic,
-  onChangeTopic,
-  selectedOffset,
-  onChangeOffsetValue,
-  confirmCheckboxChecked,
-  onConfirmationChange,
-  isSelected,
-  onSelectAll,
-  onSelectRow,
   disableFocusTrap,
   appendTo,
+  consumers,
+  state,
 }) => {
   const { t } = useTranslation(["kafka"]);
 
+  const [selectedTopic, setSelectedTopic] = useState<string>("");
+  const [selectedOffset, setSelectedOffset] = useState<OffsetValue>("absolute");
+  const [customOffsetValue, setcustomOffsetValue] = useState<string>();
+
+  const [isCheckboxChecked, setIsCheckboxChecked] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isResetAccepted, setIsResetAccepted] = useState<boolean>(false);
```

I think this makes it a bit more clear what this state is for, what do you think?

Reply:
yeah, I agree

Reply:
changes as the review comment